bin/import_categories

```diff
@@ -72,12 +71,14 @@ sub make_categories {
             category => $cat->{category}
         });
         $child_cat->email($cat->{email});
+        $child_cat->send_method("Email") if $cat->{devolved};
```

Comment:
Perhaps `$child_cat->send_method($cat->{devolved}) if $cat->{devolved};` and set that to Email so it could be something else in future?

Reply:
Nice, will do.